applications/Preprocessor/CMakeLists.txt

```diff
@@ -1,44 +1,46 @@
-include_directories(. ${hpGEM_SOURCE_DIR}/kernel/)
 
-#the preprocessor does not perform floating poing arithmetic, so it doesn't need special Petsc/not petsc guards
-add_executable(Preprocessor.out
-               main.cpp
-               centaur.cpp
-               hpgem.cpp
-               ../../kernel/Logger.cpp
-               ../../kernel/Base/CommandLineOptions.cpp
-               ../../kernel/Base/MpiContainer.cpp)
-
-add_executable(ExtractSurface.out
-               extractSurface.cpp
-               centaur.cpp
-               hpgem.cpp
-               ../../kernel/Logger.cpp
-               ../../kernel/Base/CommandLineOptions.cpp
-               ../../kernel/Base/MpiContainer.cpp)
-
-set(CURRENT_TARGET Preprocessor.out)
-include(${CMAKE_SOURCE_DIR}/conf/cmake/CLionFix.cmake)
```

Comment:
Don't we need this anymore?

Reply:
I do not know actually, I do not use CLion. :)

Reply:
I'll try then.

Reply:
Ahh you use CLion?